app/src/main/java/com/asfoundation/wallet/di/AnalyticsModule.kt

```diff
@@ -12,7 +12,7 @@ import com.asfoundation.wallet.billing.analytics.BillingAnalytics
 import com.asfoundation.wallet.billing.analytics.PageViewAnalytics
 import com.asfoundation.wallet.billing.analytics.WalletsAnalytics
 import com.asfoundation.wallet.feature_flags.topup.TopUpDefaultValueProbe
-import com.asfoundation.wallet.home.ui.HomeAnalytics
+import com.asfoundation.wallet.wallet_v3.home.HomeAnalytics
```

Comment:
I think we shouldn't have the version of the app on the name of the packages, it could be hard to change in the future